.env.example

```
@@ -9,4 +9,10 @@ GENERATE_NOTEBOOKS_BUCKET_NAME=earthdata-search-dev-generate-notebooks
 NODE_ENV=development
 
 USE_IMAGE_CACHE=false
+USE_GEOCODER=false
+USE_NLP_SEARCH=false
 SKIP_SQS=true
+
+# Change these values to enable geocoder and NLP search functionality for local development
+AWS_ACCESS_KEY_ID=CHANGE-ME
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The AWS_ACCESS_KEY_ID key is duplicated

(DuplicatedKey)